.github/workflows/main.yml

```diff
@@ -28,6 +28,10 @@ jobs:
       run: |
         sudo apt-get install clang-format
     
+    - name: Generate format guide
+      run: |
+        clang-format -style=google -dump-config > .clang-format
```

Comment:
Could we just have a committed .clang-format consisting of `BasedOnStyle:  Google` instead? That'd help people running the tool manually to format their own changes.

Reply:
Thanks for pointing this out. I'm new to clang-format and didn't realize it was so easy